README.md

```diff
@@ -2,12 +2,15 @@
 
 This a Jekyll website. **Please note that every commit in the 'gh-pages' branch will rebuild the whole website!**
 
+# Warning
```

Comment:
Done!

Reply:
Instead = "À la place", c'est le titre "warning" qui me dérange. Car il ne dit rien sur le contenu du chapitre.

Le fait que `_site` soit réécrit vient du fait que c'est un site Jekyll, ce qui est mentionné juste en dessus. Voilà ma proposition:
```
# Contribute

This a Jekyll website. **Please note that every commit in the 'gh-pages' branch will rebuild the whole website (into _site directory)**

# Kambasu 1.2.2
* Kambasu (1.2.2) is gitted in a folder because newest versions are not compatible with this website and because jekyll needs them to build.
```
J'ai seulement ajouté "(into _site directory)" à la phrase précédente.

C'est important d'être précis et court dans la documentation, ce sera plus facile à maintenir.

Reply:
ok, compris et fait.